exporter/exporter.go

```diff
@@ -64,6 +64,7 @@ func getNebulaMetrics(ipAddress string, port int32) ([]string, error) {
 	if err != nil {
 		return []string{}, err
 	}
+	defer resp.Body.Close()
 
 	bytes, err := ioutil.ReadAll(resp.Body)
```

Comment:
is it possible '\n' or '\r\n' at the end line?
is it possible non-printable in each line?
is it necessary use buff for http body?
```
	metrics := make([]string, 0)
	scanner := bufio.NewScanner(resp.Body)
	for scanner.Scan() {
		if line := strings.TrimSpace(sc.Text()); line != ""{
			metrics = append(metrics, line)
		}
	}
```

Reply:
each line end with "\n" and is printable metrics，if  metrics are large use buff more better.